src/main/java/hudson/plugins/jira/JiraProjectProperty.java

```diff
@@ -66,8 +69,10 @@ public DescriptorImpl getDescriptor() {
     @Extension
     public static final DescriptorImpl DESCRIPTOR = new DescriptorImpl();
 
+    @Symbol("jira")
```

Comment:
what is this for?

Reply:
see https://github.com/jenkinsci/configuration-as-code-plugin/pull/432/files#diff-98519df47b68dd6c5fa65a3101a8d7deL2

symbol is used to give class a "human friendly name". In JCasC context this allow to use an reasonable name for Jira configuration section